README.md

```diff
@@ -147,6 +147,62 @@ One (and only one!) of your test files needs to include _snitch_ as:
 
 See the documentation for the [header-only mode](#header-only-build) for more information. This will include the definition of `main()` [unless otherwise specified](#using-your-own-main-function).
 
+## Example build configuration with meson
+
+First, [meson build](https://mesonbuild.com/)
+needs a
+[`subprojects`](https://mesonbuild.com/Subprojects.html#using-a-subproject)
+directory in the project source root
+for dependencies, then:
+
+```bash
+> meson wrap install snitch
+```
+
+downloads a reviewed wrap-file from
+[WrapDB](https://mesonbuild.com/Wrapdb-projects.html)
+to `subprojects/snitch.wrap`  
+Here's an equivalent wrap-git specification, for illustration:
+
+```ini
+[wrap-git]
+directory = snitch
+url = https://github.com/cschreib/snitch.git
+revision = v1.1.0
```

Comment:
To mirror what is done in the CMake instructions; I don't want to have to update this number all the time ;)
```suggestion
revision = v1.1.0 ; update version number if needed
```

Reply:
I'll remove the sample wrap file and give a link to the wrap file format documentation instead.
The WrapDB file is for all but 'expert' use cases.

I'll mention that the `[provide] snitch = snitch_dep` section is important for fallback.